schema/codelists/procurementCategory.csv

```diff
@@ -1,4 +1,4 @@
 Code,Title,Description
-goods,Goods and supplies,The primary object of this contracting process involves physical or digital goods or supplies.
-works,Works,"The primary object of this contracting process involves construction, repair, rehabilitation, demolition, restoration or maintenance of some asset or infrastructure."
-services,Services,"The primary object of this contracting process involves professional services of some form, generally contracted for on the basis of measurable outputs or deliverables."
+goods,Goods and supplies,The primary object of this contracting (or planning) process involves physical or digital goods or supplies.
```

Comment:
Same concerns as for extendedProcurementCategory.

Reply:
same as above

Reply:
I've replaced "involve" with "be", e.g. "The primary object are physical or digital goods." Sounds fine by me? 

I've also opened https://github.com/open-contracting/standard/issues/1525.